sphinx/index.md

```diff
@@ -0,0 +1,43 @@
+# containerd
+
+TBD
+
+```{toctree}
+:hidden:
+:caption: Getting Started
+ops.md
+content-flow.md
+garbage-collection.md
+hosts.md
+managed-opt.md
+stream-processors.md
+```
+
+```{toctree}
+:hidden:
+:caption: Advanced
+namespaces.md
+remote-snapshotter.md
+tracing.md
+rootless.md
+```
+
+```{toctree}
+:hidden:
+:caption: Developing containerd client
+getting-started.md
+client-opts.md
+```
+
+```{toctree}
+:hidden:
+:caption: CRI
+cri/config.md
+cri/crictl.md
+cri/decryption.md
+cri/installation.md
+cri/proposal.md
```

Comment:
This is historical (https://github.com/containerd/containerd/pull/6754), no need to be linked here